src/window/action_popup.h

```diff
@@ -0,0 +1,12 @@
+#ifndef WINDOW_ACTION_DIALOG_H
+#define WINDOW_ACTION_DIALOG_H
+
+#include <stdint.h>
+#include "window/option_popup.h"
+
+void window_action_popup_show(int title, int subtitle, option_menu_item *options,
```

Comment:
Should be `const option_menu_item *options`

Reply:
k